README.md

```diff
@@ -1,90 +1,99 @@
+- A set of tools to ease common distributed app development needs (e.g. a log aggregator to ease log-diving, automatic port-forwarding to ease connectivity, a `kurtosis service shell` command to ease container filesystem exploration, etc.)
 
 Why should I use Kurtosis?
 ==========================
-Kurtosis is valuable to blockchain infrastructure developers because, when compared to alternatives, our _environment definitions_ are:
-* 🧱 **Modular** - parts of the system can be combined or be taken apart at the container level
-* ✏️ **Easy to write** - we know `.yaml` doesn't cut it, so we use a Python dialect called Starlark, written and used by Google ([Bazel](https://bazel.build/)) and Meta ([Buck](https://buck2.build/))
-* 🪜 **Sequential** - catering to the specific node-bootstrapping needs of blockchains
-* 📈 **Scale-able** - operate at any scale you need, limited only by the underlying hardware
-* 🔌 **Integratable** - plug & play integration with your existing infrastructure & made to be built on top of
-* ♻️ **Reproducable** - Completely deterministic and idempotent: given the same inputs, the system will start up the exact same way, every time
-* 🆓 **Open source** - Kurtosis is transparent and will be free, forever
-
-Still curious?
-==============
-* Kurtosis Web3 use cases [here](https://web3.kurtosis.com).
-* Learn about "why Kurtosis?" [here](https://docs.kurtosis.com/#why-use-kurtosis).
-* To read about the architecture, go [here](https://docs.kurtosis.com/explanations/architecture).
-* To get a peek at where we're going with the product, check out the roadmap [here](https://github.com/kurtosis-tech/kurtosis/wiki/Short%E2%80%90term-Roadmap).
-* For real examples, go [here](https://github.com/kurtosis-tech/awesome-kurtosis)
-
-Kurtosis Kloud Early Access
-===========================
+Kurtosis shines when creating, working with, and destroying self-contained distributed application environments. Currently, our users report this to be most useful when:
 
-If you're looking to run Kurtosis environments on the cloud, look no further! 
-We're excited to launch an early access offering for Kurtosis Kloud. Once you [sign up](https://mp2k8nqxxgj.typeform.com/to/U1HcXT1H), we'll reach out to you with the next steps.
+- You're developing on your application and you need to rapidly iterate on it
+- You want to try someone's containerized service or distributed application without setting up an environment, dependencies, etc.
+- You want to spin up your distributed application in ephemeral environments as part of your integration tests
+- You want to ad-hoc test your application on a big cloud cluster
+- You're the author of a containerized service or distributed application and you want to give your users a one-liner to try it
+- You want to get an instance of your application running in the cloud without being a Kubernetes expert
```

Comment:
Suggested rewrite of the last one:

- You want to get an instance of your application running in the cloud without dealing with provisioning hardware or administering Kubernetes